shared/actions/login/index.js

```diff
@@ -491,11 +475,12 @@ function makeKex2IncomingMap (dispatch, getState, onBack: SimpleCB, onProvisione
           onBack={() => onBack(response)} />))
     },
     'keybase.1.provisionUi.chooseDevice': ({devices}, response) => {
+      const d = devices || []
       appendRouteElement((
         <SelectOtherDevice
           devices={devices}
           onSelect={deviceID => {
-            const type: DeviceRole = devices[devices.findIndex(d => d.deviceID === deviceID)].type
+            const type: DeviceRole = d[d.findIndex(d => d.deviceID === deviceID)].type
```

Comment:
we don't handle that well now. unlikely to be an issue as you've clicked an item in the list and we're just plucking extra data out